### ISSUE TITLE
Gameplay: it's almost too easy to start from the edges

### ISSUE THREAD

**Issue by stevegrossi** [stevegrossi/sengoku#13]
Similar to Risk's "Australia first" strategy, in playing against the computer it's almost always the best choice to start by conquering the far left or far right of the map, because the advantage of only having one border to defend is too great. This makes the game less interesting. How can I make capturing (and holding) the middle of the map a more appealing option?

#12 may help: with the introduction of bonuses for holding regions, I can assign larger bonuses to the middle regions. I can also tweak the region demarcations a bit to make middle regions easier to hold (fewer perimeter tiles).

**Comment by yakryder** [comment]
You can incentivize holding the center as you suggest. But you could also penalize holding the edges

**Comment by stevegrossi** [comment]
In #12 I did a little bit of both. Bigger bonuses for holding the middle of the map, and I added an extra connection to the leftmost island so it's no longer an "Australia" with just one way in or out. Anecdotally, the game feels a little more interesting now playing against the AI, especially now that they attempt to conquer regions. The penalty for focusing on the edges as a human player is that the AI will conquer it and the large unit bonuses mean the AI may win as a result.

I'll close this issue for now but am definitely interested in continuing to tweak as a result of feedback.
